README.md

```diff
@@ -32,12 +30,32 @@ There is also a Slack channel for discussion and support. Click [here](https://t
 
 ## Development
 
-To install this extension in editable mode, simply change to this directory, and run:
+First setup your virtual environment and install required dependencies:
+
+```sh
+python -m venv .venv
```

Comment:
See https://github.com/CQCL/pytket-qir/pull/57#discussion_r1280635199

I think it would be good to do something like

```
First install required dependencies:

```sh
pip install -U pip setuptools
pip install build pre-commit pytest wheel mypy~=1.4 black~=23.7 pylint~=2.17 ruff==0.0.282
pre-commit install
```

Then install this extension in editable mode, simply change to this directory, and run:

Reply:
What's the argument against providing steps for creating a virtual environment for beginners? I like to provide a self-contained set of steps (especially those that I can copy and paste myself without having to think). People who are already familiar with virtual environments and prefer setting them up any other way can just skip copying the first line, no?